### PR TITLE
Homework #4 Tests

### PR DESCRIPTION
**Pull request by lesnata**

### CODE REVIEW

File: tests/tests.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        self.assertEqual(self.test_Rectangle1.get_radius_of_inscribed_circle(), result)
+
+    def test_get_radius_of_inscribed_circle_equality(self):
+        if Rectangle.width == Rectangle.height:
+            self.assertRaises(ValueError, self.test_Rectangle1.get_radius_of_inscribed_circle)
+
+
+if __name__ == '__main__':
+    unittest.main()
+
```
Comment (alchemixXx) [inline tests/tests.py]: Згідно РЕР8, в кінці файлу лише 1 пуста строка

File: tests/tests.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+import unittest
+import sys
+from math import sqrt
+
+from homework import Rectangle
+
+
+class TestCases(unittest.TestCase):
+
+    def tested_values(self):
```
Comment (alchemixXx) [inline tests/tests.py]: better to use setUp

File: tests/tests.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    def test_get_rectangle_square_2_empty_sides(self):
+        self.assertIsNone(self.test_Rectangle2.get_rectangle_square())
+
+    def test_get_sum_of_corners_0_to_3(self):
+        for i in range(4):
+            self.assertEqual(self.test_Rectangle1.get_sum_of_corners(i), i*90)
+
+    def test_get_sum_of_corners_4_more(self):
+        for i in range(5, sys.maxsize):
```
Comment (alchemixXx) [inline tests/tests.py]: loop with sys.maxsize - could fulfill you RAM. Put regular number here

File: tests/tests.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    def test_get_rectangle_perimeter_2_empty_sides(self):
+        self.assertIsNone(self.test_Rectangle2.get_rectangle_perimeter())
+
+    def test_get_rectangle_square_1_valid_values(self):
+        self.assertEqual(self.test_Rectangle1.get_rectangle_square(), 12.5)
+
+    def test_get_rectangle_square_2_empty_sides(self):
+        self.assertIsNone(self.test_Rectangle2.get_rectangle_square())
+
+    def test_get_sum_of_corners_0_to_3(self):
```
Comment (alchemixXx) [inline tests/tests.py]: 4 - still good value, isn't it?

File: tests/tests.py (hunk 1 above)
Comment (lesnata) [reply tests/tests.py]: done

**Pull request merged**